auto_revoke_sg_rules/main.tf

```diff
@@ -0,0 +1,85 @@
+/* 
+* Automatically revoke security group changes on SSH and RDP ports (22 and 3389) 
+*
+* This module sets up a lambda that will automatically revert any security group changes that open the SSH and RDP ports.
+*/
+
+# Get the current AWS account ID and region
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+# Lambda: Disable secruity group exposure to ports 22 and 3389 
+#
+data "archive_file" "sg_change_auto_response" {
+  type        = "zip"
+  source_file = "${path.module}/functions/sg_change_auto_response.py"
+  output_path = "/tmp/sg_change_auto_response.py.zip"
+}
+
+# Lambda function that is used for security group auto-response
+resource "aws_lambda_function" "security_group_change_auto_response" {
+  function_name = "security_group_change_auto_response"
```

Comment:
Consider making this a variable with the current value as the default. Here is an example

https://github.com/cds-snc/terraform-modules/blob/main/sentinel_forwarder/main.tf#L35

and here is the input

https://github.com/cds-snc/terraform-modules/blob/main/exposed_iam_credentials_disabler/input.tf#L13-L27

This could avoid potential namespace collisions, however unlikely.

Reply:
Thanks for the suggestion Max - I have made the change.